### PR TITLE
feat: add blog post for 12 week release cadence

### PR DESCRIPTION
**Pull request by sofianguy**
Request for copy-editing please 🙏 

TODO: edit publish date before merge

### CODE REVIEW

File: data/blog/12-week-cadence.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+date: '2019-05-14'
+---
+
+🎉 Electron is moving to release a new stable version every 12 weeks! 🎉
+
+---
+
+## ⚡️ Wow that's quick! But why?
+
+Simply put, Chromium doesn't stop shipping so Electron is not going to slow down either.
```
Comment (codebytere) [inline data/blog/12-week-cadence.md]: We should probably provide more contextualizing information here. For example:

> Chromium releases on a consistent 6-week schedule, and so in order to continue to deliver the most up-to-date versions of Chromium in Electron we need to coordinate our schedule to match theirs. More information around Chromium's release cycle can be found [here](https://chromium.googlesource.com/chromium/src/+/master/docs/process/release_cycle.md).

File: data/blog/12-week-cadence.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## 🚀 Why is 12 weeks the chosen cadence?
+
+Right now we are planning to upgrade every other Chromium version. Chromium ships a new stable version at a very quick pace of every 6 weeks. We have adjusted our stable release dates to match every other Chromium stable release. Up first, Electron v6.0.0 will include M76 and is scheduled for stable release on [July 30, 2019](https://electronjs.org/docs/tutorial/electron-timelines#600-release-schedule), the same release day as [Chromium M76](https://www.chromium.org/developers/calendar).
```
Comment (codebytere) [inline data/blog/12-week-cadence.md]: We might want to link to M76 specifically here

File: data/blog/12-week-cadence.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## 🚧 What are the side effects of this new release cadence?
+
+We recommend downstream apps to start being more conscious of Electron upgrades. [Our policy](https://electronjs.org/docs/tutorial/support#supported-versions) to support the latest three major versions will remain unchanged. For example, when [v6.0.0 goes stable on July 30, 2019](https://electronjs.org/docs/tutorial/electron-timelines#600-release-schedule) that means we will support v6.x, v5.x, and v4.x and v3.x will become end-of-life.
```
Comment (codebytere) [inline data/blog/12-week-cadence.md]: > We recommend downstream apps to start being more conscious of Electron upgrades.

(nit) this is probably better phrased as:
> We recommend that our downstream consumers begin to plan further in advance for Electron upgrades as they are released in accordance with this new schedule.

Comment (codebytere) [reply data/blog/12-week-cadence.md]: > v6.x, v5.x, and v4.x and v3.x will become end-of-life.

 is perhaps clearer as:

> v6.x, v5.x, and v4.x, while v3.x will reach End-Of-Life.

File: data/blog/12-week-cadence.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## 📝 A bit of history on Electron releases
+
+The decisions around stable releases before v3.0.0 were unplanned and unpredictable. We set release dates internally for v3.0.0 and v4.0.0. Earlier this year, we decided to publicize our stable release date for the first time for [Electron v5.0.0](https://electronjs.org/blog/electron-5-0-timeline). Announcing our stable release dates was positively received overall and we're excited to continue doing that for future releases.
+
+After we implemented [Electron governance](https://electronjs.org/blog/governance) a few months ago, Electron maintainers have more structure in the day-to-day work. The [Upgrades Working Group](https://github.com/electron/governance/tree/master/wg-upgrades) and [Releases Working Group](https://github.com/electron/governance/tree/master/wg-releases) are working together to get us catching up to Chromium, starting with our 12 week release cadence.
```
Comment (codebytere) [inline data/blog/12-week-cadence.md]: > After we implemented [Electron governance](https://electronjs.org/blog/governance) a few months ago, Electron maintainers have more structure in the day-to-day work.

Perhaps:

> In order to better streamline these upgrade-related efforts, our Upgrades and Releases Working Groups (add links) were created within our Governance (add link) system. They have allowed us to better prioritize and delegate this work, which we hope will become more apparent with each subsequent release.

File: data/blog/12-week-cadence.md (hunk 2 above)
Comment (sofianguy) [reply data/blog/12-week-cadence.md]: Do know a link for M76? I tried looking for a specific link but couldn't find any 🤔

Comment (codebytere) [reply data/blog/12-week-cadence.md]: i'd say this might be better: https://www.chromestatus.com/features/schedule

Comment (sofianguy) [reply data/blog/12-week-cadence.md]: perfect! thank you

File: data/blog/12-week-cadence.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+🎉 Electron is moving to release a new stable version every 12 weeks! 🎉
+
+---
+
+## ⚡️ Wow that's quick! But why?
+
+Simply put, Chromium doesn't stop shipping so Electron is not going to slow down either.
+
+Chromium releases on a consistent 6-week schedule, and so in order to continue to deliver the most up-to-date versions of Chromium in Electron we need to coordinate our schedule to match theirs. More information around Chromium's release cycle can be found [here](https://chromium.googlesource.com/chromium/src/+/master/docs/process/release_cycle.md).
```
Comment (ckerr) [inline data/blog/12-week-cadence.md]:
```suggestion
Chromium releases on a consistent 6-week [schedule](https://www.chromium.org/developers/calendar). To deliver the most up-to-date versions of Chromium in Electron, our schedule needs to track theirs. More information around Chromium's release cycle can be found [here](https://chromium.googlesource.com/chromium/src/+/master/docs/process/release_cycle.md).
```

File: data/blog/12-week-cadence.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+---
+title: New Electron Release Cadence
+author: sofianguy
+date: '2019-05-14'
+---
+
+🎉 Electron is moving to release a new stable version every 12 weeks! 🎉
```
Comment (ckerr) [inline data/blog/12-week-cadence.md]:
```suggestion
🎉 Electron is moving to release a new major stable version every 12 weeks! 🎉
```

File: data/blog/12-week-cadence.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+---
+
+## ⚡️ Wow that's quick! But why?
+
+Simply put, Chromium doesn't stop shipping so Electron is not going to slow down either.
+
+Chromium releases on a consistent 6-week schedule, and so in order to continue to deliver the most up-to-date versions of Chromium in Electron we need to coordinate our schedule to match theirs. More information around Chromium's release cycle can be found [here](https://chromium.googlesource.com/chromium/src/+/master/docs/process/release_cycle.md).
+
+## 🚀 Why is 12 weeks the chosen cadence?
```
Comment (ckerr) [inline data/blog/12-week-cadence.md]:
```suggestion
## 🚀 Why every 12 weeks?
```

File: data/blog/12-week-cadence.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+Right now we are planning to upgrade every other Chromium version. Chromium ships a new stable version at a very quick pace of every 6 weeks. We have adjusted our stable release dates to match every other Chromium stable release. Up first, Electron v6.0.0 will include M76 and is scheduled for stable release on [July 30, 2019](https://electronjs.org/docs/tutorial/electron-timelines#600-release-schedule), the same release day as [Chromium M76](https://www.chromestatus.com/features/schedule).
```
Comment (ckerr) [inline data/blog/12-week-cadence.md]:
```suggestion
Every 6 weeks, a new Chromium release comes out with new features, bugfixes / security fixes, and V8 improvements. Electron's users have been loud and clear about wanting these changes in a timely manner, so we've adjusted our stable release dates to match every other Chromium stable release. Up first, Electron v6.0.0 will include M76 and is scheduled for stable release on [July 30, 2019](https://electronjs.org/docs/tutorial/electron-timelines#600-release-schedule), the same release day as [Chromium M76](https://www.chromestatus.com/features/schedule).
```

File: data/blog/12-week-cadence.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+## 🚧 What are the side effects of this new release cadence?
```
Comment (ckerr) [inline data/blog/12-week-cadence.md]:
```suggestion
## 🚧 What does this mean for me and my Electron app?
```

File: data/blog/12-week-cadence.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+
+We recommend that our downstream consumers begin to plan further in advance for Electron upgrades as they are released in accordance with this new schedule. [Our policy](https://electronjs.org/docs/tutorial/support#supported-versions) to support the latest three major versions will remain unchanged. For example, when [v6.0.0 goes stable on July 30, 2019](https://electronjs.org/docs/tutorial/electron-timelines#600-release-schedule) that means we will support v6.x, v5.x, and v4.x, while v3.x will reach End-Of-Life.
```
Comment (ckerr) [inline data/blog/12-week-cadence.md]:
```suggestion
You'll have access to new Chromium and V8 features and fixes sooner than before. Importantly, you'll also know _when_ those new changes are coming, so you'll be able to plan with better information than before.

The Electron team will [continue to support](https://electronjs.org/docs/tutorial/support#supported-versions) the latest three major versions. For example, when [v6.0.0 goes stable on July 30, 2019](https://electronjs.org/docs/tutorial/electron-timelines#600-release-schedule), we will support v6.x, v5.x, and v4.x, while v3.x will reach End-Of-Life.
```

File: data/blog/12-week-cadence.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+
+## 💬 App Feedback Program
+
+Please consider joining our [App Feedback Program](https://electronjs.org/blog/app-feedback-program) to help us with testing our beta releases and stabilization. Projects who participate in this program test Electron betas on their apps; and in return, the new bugs they find are prioritized for the stable release.
+
+## 📝 A bit of history on Electron releases
```
Comment (ckerr) [inline data/blog/12-week-cadence.md]:
```suggestion
## 📝 A brief history of Electron releases
```

File: data/blog/12-week-cadence.md [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+
+The decisions around stable releases before v3.0.0 were unplanned and unpredictable. We set release dates internally for v3.0.0 and v4.0.0. Earlier this year, we decided to publicize our stable release date for the first time for [Electron v5.0.0](https://electronjs.org/blog/electron-5-0-timeline). Announcing our stable release dates was positively received overall and we're excited to continue doing that for future releases.
```
Comment (ckerr) [inline data/blog/12-week-cadence.md]:
```suggestion
The decisions around stable releases before v3.0.0 did not follow a schedule. We added internal schedules to the project with v3.0.0 and v4.0.0. Earlier this year, we decided to publicize our stable release date for the first time for [Electron v5.0.0](https://electronjs.org/blog/electron-5-0-timeline). Announcing our stable release dates was positively received and we're excited to continue doing that for future releases.
```

File: data/blog/12-week-cadence.md [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+
+In order to better streamline these upgrade-related efforts, our [Upgrades](https://github.com/electron/governance/tree/master/wg-upgrades) and [Releases](https://github.com/electron/governance/tree/master/wg-releases) Working Groups were created within our [Governance](https://electronjs.org/blog/governance) system. They have allowed us to better prioritize and delegate this work, which we hope will become more apparent with each subsequent release.
```
Comment (ckerr) [inline data/blog/12-week-cadence.md]: :+1:

File: data/blog/12-week-cadence.md [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+
+Take a look at the graph below to see where this new cadence will put us in comparison to Chromium's cadence.
```
Comment (ckerr) [inline data/blog/12-week-cadence.md]:
```suggestion
Here is where our new cadence will put us in comparison to Chromium's cadence:
```

**Pull request merged**